onebusaway-android/src/main/java/org/onebusaway/android/report/ui/Open311ProblemFragment.java

```diff
@@ -492,16 +498,27 @@ private String getTransitIssueParameters(Service service) {
             // Append trip service params
             sb.append(getResources().getString(R.string.ri_append_stop_id, obaStop.getId()));
             if (mArrivalInfo != null) {
+                DateFormat dateFormat = new SimpleDateFormat("MM-dd-yyyy");
+
                 sb.append(getResources().getString(R.string.ri_append_trip_name, mArrivalInfo.getHeadsign()));
                 sb.append(getResources().getString(R.string.ri_append_route_id, mArrivalInfo.getRouteId()));
+                if (!TextUtils.isEmpty(mArrivalInfo.getShortName())) {
+                    sb.append(getResources().getString(R.string.ri_append_route_short_name,
+                            mArrivalInfo.getShortName()));
+                } else if (!TextUtils.isEmpty(mArrivalInfo.getRouteLongName())) {
+                    sb.append(getResources().getString(R.string.ri_append_route_long_name,
+                            mArrivalInfo.getRouteLongName()));
+                }
```

Comment:
Thinking more about this - it's probably better to create a new UIUtils method that takes in an ArrivalInfo object, and then refactor UIUtils.getRouteDisplayName(ObaRoute route) and this new method to both use another new method UIUtils.getRouteDisplayName(String routeShortName, String routeLongName), and returns the correct name.  This way you don't have to pull out a ObaRoute object when you already have the same info in the ArrivalInfo object.